postgresql/Dockerfile

```diff
@@ -14,13 +14,8 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/*
 
 ADD *.sql /docker-entrypoint-initdb.d/
+# This script is used to skip loading of sample data if IGNORE_DATA var is set to yes
+ADD 000-skip_data.sh /docker-entrypoint-initdb.d/
 ADD license.txt /docker-entrypoint-initdb.d/
 ADD logo.png /docker-entrypoint-initdb.d/
```

Comment:
Instead of several ADD instructions, use something like https://github.com/camptocamp/georchestra-bootstrap-volumes/blob/master/Dockerfile#L16